client/apps/edit2/components/admin/components/verticals_tags.jsx

```diff
@@ -1,18 +1,20 @@
 import PropTypes from 'prop-types'
 import React, { Component } from 'react'
 import { Col, Row } from 'react-styled-flexboxgrid'
-import { data as sd } from 'sharify'
+import { connect } from 'react-redux'
 import Verticals from '../../../../../collections/verticals.coffee'
 import { AutocompleteInlineList } from '/client/components/autocomplete2/inline_list'
+import { onChangeArticle } from 'client/actions/editActions'
 
 export class AdminVerticalsTags extends Component {
   static propTypes = {
+    apiURL: PropTypes.string,
```

Comment:
Curious why you want to make this into a prop instead of pulling from Sharify?

Reply:
Really could go either way -- some of these urls worked their way into the store during the lockout project, which also meant they were available within the redux actions without having to loop in sd.  Since they were already there, I started using them rather that requiring another library.

Reply:
they are in the app reducer: https://github.com/artsy/positron/blob/master/client/reducers/appReducer.js